aws/backbone/README.md

```diff
@@ -52,6 +52,18 @@ This module sets up an AWS backbone base infrastructure.
   VPC CIDR block.
   - default: none
 
+- `vgw_id` (string)
+
+  Id ot the virtual gateway to attach to the VPC (`""`: no VGW)
```

Comment:
Nitpick: Id o**F**
Also, keeping your idea from the description, wouldn't it be better to write `No attachment if empty` here?

Reply:
Yes sure